font/src/lib.rs

```diff
@@ -267,3 +321,27 @@ pub trait Rasterize {
     /// Update the Rasterizer's DPI factor
     fn update_dpr(&mut self, device_pixel_ratio: f32);
 }
+
+/// Config option specific to the Rasterizer.
+/// Since the Rasterizer lives in the subcrate font we do not want it to depend on the Font config
+/// struct from alacritty, as this would introduce a circular dependency between the crates.Clone
+/// This struct specifies the subset of Font that the Rasterizer cares about, then traits can be
+/// used to convert from Font to this struct when constructing a Rasterizer.
+#[cfg(feature = "hb-ft")]
+pub struct RasterizeConfig {
+    /// Toggle thin strokes on mac osx
+    // Technically this is impossible while under "hb-ft" but is included for compatiblity in the
+    // api
```

Comment:
If it is impossible, we should remove `Technically`. If there is a way to somehow do this, we should implement it.

I'd also be curious if there's any upstream tracking issue with freetype for this. Is this not relevant for harfbuzz at all? Then we probably don't even need it, right? In which case we could also just pass `use_font_ligatures` alone without the `RasterizeConfig` struct?